index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Jauhari Alafi</title>
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <!-- CSS -->
+    <link rel="stylesheet" href="styles/global.css" />
+    <link rel="stylesheet" href="styles/template.css" />
+    <link rel="stylesheet" href="styles/main.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <a href="#" class="nav-logo">Welcome</a>
+        <a href="#" class="nav-toggle"
+          ><img src="images/icons/hamburger-menu.svg" alt="Menu"
+        /></a>
+      </nav>
+    </header>
+    <section class="headline">
+      <div class="headline-primary">
+        <h2>
+          <span class="dark-blue">Hey there. I'm Mariane</span>
+          <br />
+          <span class="green">I'm a software developer</span>
+        </h2>

```

Comment:
- [ ] Please use `<h1>` heading here. H1s are a good indicator of what the most important text on a page is. As they are usually the most visually notable content on the page, a good H1 can ensure clarity for the reader. And they are also important for SEO.

Reply:
Thank you for the insight. In the Figma template, it's suggested to use `<h2>`, but yeah, I agree, `<h1>` makes more sense. I have changed it now.